### DISCUSSION TITLE
64 bit Bullseye

### DISCUSSION THREAD

**Discussion by hyperloop512** [KenT2/pipresents-gtk#2] [Q&A]
Does anyone know if the wonderful PiPresents works under 64 bit OS - or what breaks?
Thx!

**Comment by KenT2** [comment]
I have briefly tried Pi Presents GTK on Bookworm 64 bit by running through some of the examples. It seems to work OK. I also checked that the modules that are installed using PIP are 64 bit agnostic. Don't know about Bullseye, maybe you meant Bookworm.

**Comment by hyperloop512** [comment]
Thanks - I tried it on the 64 bit, but the editor didn't work (PiPresents ran fine), reinstalled the 32 bit OS, which also didn't work -- and then realised that it is an error documented in the manual with the default browser that has to be reset to chrome. Did that and all good on 32 bit at least.
